nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,37 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
```

Comment:
I think naming here is a little confusing. wrappedPkgs vs wrappedPackages seems like they should be the same thing (although I suppose it's not any worse than wrappedBins vs wrappedBinaries).